code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
```

Comment:
I don't have an issue with it so people know what the fuck the locations are however a better solution would be the things that the carp, alium and blob event uses can't remember the name right off the top of my head